lib/entities/context/DutchQuoteContext.ts

```diff
@@ -178,7 +185,13 @@ export class DutchQuoteContext implements QuoteContext {
 
   // transform a classic quote into a synthetic dutch quote
   // if it makes sense to do so
-  getSyntheticQuote(classicQuote?: Quote, routeBackToNative?: Quote): DutchQuote | null {
+  async getSyntheticQuote(classicQuote?: Quote, routeBackToNative?: Quote): Promise<DutchQuote | null> {
```

Comment:
Might be wrong but I think we'll still only return the synthetic quote if the request has [useSynethticQuotes](https://github.com/Uniswap/unified-routing-api/blob/e309e18c48cd675097da9df6d0089ff77612874f/lib/entities/context/DutchQuoteContext.ts#L103). Should we have some other kind of param or override switch as a constant somewhere? Feel like it will still be useful to force synthetic regardless of synthetic token switch.

Reply:
ended up going with semantics where _one of_ useSyntheticQuotes _or_ the synth switch are true. So they can both override each other